### ISSUE TITLE
betterprivacy missing

### ISSUE THREAD

**Issue by pirate486743186** [JustOff/ca-archive#27]
the addon betterprivacy is missing
https://web.archive.org/web/20170627200456/https://addons.mozilla.org/en-US/firefox/addon/betterprivacy/

**Comment by grahamperrin** [comment]
Yep, a blank at: 

- caa:addon/betterprivacy

https://addons.mozilla.org/addon/betterprivacy/

> This add-on has been removed by its author.

– and in the Wayback Machine, the last en-US capture seems to be as shown above with reference to (for Linux): 

- `https://addons.mozilla.org/firefox/downloads/file/528039/betterprivacy_signed-1.77-fx-linux.xpi`

**Comment by grahamperrin** [comment]
Via commentary at <https://redd.it/9b2yyb>: 

- https://github.com/xtraeme-xt/BetterPrivacy

> BetterPrivacy 1.77 before it was taken off AMO

NB presumably not the original developer https://addons.mozilla.org/en-GB/firefox/user/Netcat/ a.k.a. IKRG

- http://www.mediafire.com/file/mrqhbub97vcg9li/%2528XUL%2529_betterprivacy_signed-1.77.xpi/file

```text
$ sha256 ./\(XUL\)\ betterprivacy_signed-1.77.xpi 
SHA256 (./(XUL) betterprivacy_signed-1.77.xpi) = 99309428870cf6cfe9da91de888b3d03035ddcaaab44ca3242d3c985d1824528
$ 
```

– but I don't have an original against which the checksum might be compared.

**Comment by JustOff** [comment]
> the addon betterprivacy is missing

This add-on has been removed by the author before ca-archive backup was created.

> http://www.mediafire.com/file/mrqhbub97vcg9li/%2528XUL%2529_betterprivacy_signed-1.77.xpi/file
> `SHA256 (./(XUL) betterprivacy_signed-1.77.xpi) = 99309428870cf6cfe9da91de888b3d03035ddcaaab44ca3242d3c985d1824528`

From https://web.archive.org/web/20170627200456/https://addons.mozilla.org/en-US/firefox/addon/betterprivacy/ page source:

`<a class="button prominent platform windows add installer" data-hash="sha256:99309428870cf6cfe9da91de888b3d03035ddcaaab44ca3242d3c985d1824528" href="/web/20170627200456/https://addons.mozilla.org/firefox/downloads/latest/betterprivacy/platform:5/addon-6623-latest.xpi?src=dp-btn-primary">
  <b></b>
  <span>Add to Firefox</span>
</a>`

So this copy can be trusted.

**Comment by grahamperrin** [comment]
Interesting, what address?

I tried platform 5 `http://web.archive.org/web/20170627200456/https://addons.mozilla.org/firefox/downloads/latest/betterprivacy/platform:5/addon-6623-latest.xpi` and platform 1, neither is a capture ...

**Comment by pirate486743186** [comment]
Add "?src=dp-btn-primary" at the end.

**Comment by grahamperrin** [comment]
OK, http://web.archive.org/web/20170627200456/https://addons.mozilla.org/firefox/downloads/latest/betterprivacy/platform:5/addon-6623-latest.xpi?src=dp-btn-primary

> Got an HTTP 302 response at crawl time
> 
> Redirecting to...
> 
> https://addons.cdn.mozilla.net/user-media/addons/6623/betterprivacy_signed-1.74-fx-windows.xpi?filehash=sha256%3Af1f1ccd919b162dc31d75c4cb453292d97f53822f75685ba7d4957b6e4230875

-- captured at: 

https://web.archive.org/web/20160915014724/https://addons.cdn.mozilla.net/user-media/addons/6623/betterprivacy_signed-1.74-fx-windows.xpi?filehash=sha256%3Af1f1ccd919b162dc31d75c4cb453292d97f53822f75685ba7d4957b6e4230875

Interesting (for me) because I don't ever recall finding an `.xpi` captured from AMO before now. 

Thanks (and apologies for more ... trying to get my head around this all whilst limited to a hand-held)

**Comment by pirate486743186** [comment]
I was surprised too.... XD
I thought they were  cutting corners to save costs.
It worked with larger addons too. ~4MB

**Comment by Atavic** [comment]
So has this issue been solved?

**Comment by JustOff** [comment]
It will be closed when it solved. At the moment I don't have a defined timeline for [restoring extensions missed in CAA](https://github.com/JustOff/ca-archive/labels/data%20restore).

**Comment by JustOff** [comment]
tag #43

**Comment by JustOff** [comment]
Resolved by [CAA 2.0.1](https://github.com/JustOff/ca-archive/releases/tag/2.0.1).
